docs/interfaces/espnow.md

```
@@ -0,0 +1,66 @@
+| 15 | Cool white |
+| 16 | Red |
+| 17 | Green |
+| 18 | Blue |
+| 19 | Yellow |
+| 20 | Candle effect |
+| 21 | Random colour |
+| 22 | Fade effect |
+
+Codes 20–22 were added in v16.0
+
+Additional codes can be used by adding a `remote.json` file. There are some examples on how to do that available [here](https://github.com/DedeHai/WLED-ESPNow-Remote/tree/main/remote-json).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)